partner_accreditation/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Add Accreditation to Partners
```

Comment:
A brief explanation of what Accreditation for a Partner is may help user to understand the benefit of this module.

Reply:
added :)